src/gui/vector/qgswmsdimensiondialog.cpp

```diff
@@ -147,8 +147,13 @@ void QgsWmsDimensionDialog::nameChanged( const QString &name )
     int data = mNameComboBox->currentData().toInt();
     if ( data == QgsVectorLayerServerProperties::TIME )
     {
-      mFieldComboBox->setFilters( QgsFieldProxyModel::String | QgsFieldProxyModel::DateTime );
-      mEndFieldComboBox->setFilters( QgsFieldProxyModel::String | QgsFieldProxyModel::DateTime );
+      QgsFieldProxyModel::Filters filters = QgsFieldProxyModel::String |
```

Comment:
```suggestion
      const QgsFieldProxyModel::Filters filters = QgsFieldProxyModel::String |
```

Reply:
Thanks I have think about it and forgot to do it.